packages/eslint-config/index.js

```diff
@@ -1,4 +1,12 @@
-const prettierConfig = require('./prettier.config');
+// The following are ESLint rules that collide with our Prettier config. If we don't disable these
+// then code formatted by Prettier will cause ESLint to throw errors.
+const prettierOverrides = {
```

Comment:
i _think_ this should do the trick:

```suggestion
/** @type {Partial<import("eslint-define-config").Rules>} */
const prettierOverrides = {
```